src/Sylius/Component/Core/Model/Product.php

```diff
@@ -203,6 +203,10 @@ public function hasTaxon(BaseTaxonInterface $taxon)
      */
     public function getPrice()
     {
+        if (null === $this->getMasterVariant()) {
+            return 0;
```

Comment:
No. It is written this way in purpose, without it I wasn't able to present update page.